src/Bridge/Symfony/Routing/RouteNameGenerator.php

```diff
@@ -44,18 +45,39 @@ private function __construct()
      *
      * @return string
      */
-    public static function generate(string $operationName, string $resourceShortName, $operationType): string
+    public static function generate(string $operationName, string $resourceShortName, $operationType, string $property = null, bool $subresourceCollection = false): string
```

Comment:
ping @meyerbaptiste I have to add `property` and `collection` information everywhere :|. 

I'm not satisfied with this, WDYT? 

I haven't changed the `OperationPathResolver`s yet because I'd need to add so many hacks to add those 2 new arguments... Let me know if you've a better idea in mind, I'm kinda stuck in the no-breaking-change-loop here :smile:.